eslint.config.js

```diff
@@ -129,7 +129,7 @@ module.exports = [
     },
   },
   {
-    files: ['**/*.mocha.js'],
+    files: ['**/*.mocha.js', 'test/webdriverio/test/*_test.mjs'],
```

Comment:
Alternatively we could rename `basic_test.mjs` to `basic.mocha.mjs` and just tweak the pattern to add the `mjs` extension.